tools/background_removal/background_removal.xml

```diff
@@ -0,0 +1,55 @@
+<tool id="background_removal" name="Background Removal" version="0.1">
```

Comment:
```suggestion
<tool id="background_removal" name="Remove image background" version="0.1">
```
This is more in line with [our guidelines](https://osf.io/preprints/biohackrxiv/w8dsz)
> To maintain consistency, we consider phrasing names as imperatives a good practice, such as “Analyze particles” or “Perform segmentation using watershed transformation”.